content/src/cloud/troubleshooting-cloud-backup/backup-local-storage-full.md

```diff
@@ -6,12 +6,16 @@ zendesk:
   labels: cloud
 ---
 
-**Local storage is full** - If your local storage is full, or even nearing full, it may not be possible to generate the backup to upload online. Check if you have enough space in the [storage settings](https://my.home-assistant.io/redirect/storage/) to generate your backup.
+## Symptom
+
+You have [Home Assistant Cloud backup enabled](/hc/en-us/articles/26294320337181-Enabling-a-cloud-backup) in Home Assistant. You see a message stating that the **Local storage is full**.
+
+## Resolution
+
+If your local storage is full, or even nearing full, it may not be possible to generate the backup to upload online. Check if you have enough space in the [storage settings](https://my.home-assistant.io/redirect/storage/) to generate your backup.
```

Comment:
How is this a resolution?
This only confirms the problem.

https://www.home-assistant.io/more-info/free-space contains some tips on how to deal with low space

Reply:
Excellent observation.. will fix, thx